### PR TITLE
Add USDT KAVAevm

### PR DESCRIPTION
**Pull request by cryptomanz1**

### CODE REVIEW

File: blockchains/kavaevm/assets/0x919C1c267BC06a7039e03fcc2eF738525769109c/info.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+    "name": "Tether",
+    "symbol": "USDT",
+    "type": "KAVAERC20",
```
Comment (vcoolish) [inline blockchains/kavaevm/assets/0x919C1c267BC06a7039e03fcc2eF738525769109c/info.json:4]: KAVAEVM

**Review by vcoolish — APPROVED** [review]

File: blockchains/cryptoorg/assets/0xf78a326ACd53651F8dF5D8b137295e434B7c8ba5/info.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
     "name": "Matic Token",
     "symbol": "MATIC",
-    "type": "CRC20",
+    "type": "CRYPTOORG",
```
Comment (defisaur) [inline blockchains/cryptoorg/assets/0xf78a326ACd53651F8dF5D8b137295e434B7c8ba5/info.json:4]: @cryptomanz1 @vcoolish please review should it be `CRC20` or `CRYPTOORG`

Comment (vcoolish) [reply blockchains/cryptoorg/assets/0xf78a326ACd53651F8dF5D8b137295e434B7c8ba5/info.json:4]: CRC20

Comment (defisaur) [reply blockchains/cryptoorg/assets/0xf78a326ACd53651F8dF5D8b137295e434B7c8ba5/info.json:4]: @vcoolish We already have CRC20 for Cronos. Which type should be Cronos tokens?

Comment (vcoolish) [reply blockchains/cryptoorg/assets/0xf78a326ACd53651F8dF5D8b137295e434B7c8ba5/info.json:4]: not quite understand. For this kind of cryptoorg should be CRC20
for cryptoorg which is cosmos sdk based should be CRO I suppose, but we don't support it in apps.

**Comment by defisaur** [comment]
@cryptomanz1 @vcoolish pipelines are green now, but wait don't merge, we need to check something on backend due to assets moved

**Comment by defisaur** [comment]
@vcoolish @cryptomanz1 USDT KAVAevm all fixed, you can merge

As discussed in DM I also noticed we have tokens in Cryptoorg folder which should be actually in Cronos, backend-team will do moving-related works separately to not block this PR (need to think about affected parts after tokens moved, e.g. tx history and so on, [task here](https://github.com/trustwallet/backend/issues/2561))

File: blockchains/kavaevm/assets/0x919C1c267BC06a7039e03fcc2eF738525769109c/info.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
+{
+    "name": "Tether",
+    "symbol": "USDT",
+    "type": "KAVAEVM",
```
Comment (defisaur) [inline blockchains/kavaevm/assets/0x919C1c267BC06a7039e03fcc2eF738525769109c/info.json:4]: @vcoolish @cryptomanz1 KAVAEVM as requested

**Review by jamiegreen7 — COMMENTED** [review]
Review Changes

**Comment by jamiegreen7** [comment]
> ### PR Summary
> file is not allowed: go.mod: Please revert it.
> file is not allowed: go.sum: Please revert it.
> file is not allowed: internal/processor/fixers.go: Please revert it.
> Token in PR: **KAVAERC20** 0x919C1c267BC06a7039e03fcc2eF738525769109c
> Symbol: **USDT** decimals: 6
> Links: [Logo](https://raw.githubusercontent.com/trustwallet/assets/add-usdt-kavaevm/blockchains/kavaevm/assets/0x919C1c267BC06a7039e03fcc2eF738525769109c/logo.png) [Info](https://raw.githubusercontent.com/trustwallet/assets/add-usdt-kavaevm/blockchains/kavaevm/assets/0x919C1c267BC06a7039e03fcc2eF738525769109c/info.json) [Website](https://tether.to) [Explorer](https://explorer.kava.io/token/0x919C1c267BC06a7039e03fcc2eF738525769109c)
> Tags: stablecoin
> Token check error: 
> ❌ invalid type field: unknown asset type: KAVAEVM
> ❌ unknown asset type: KAVAEVM
> ❌ number of holders not checked: please, check it manually
> 
> 
> 
> <span style="padding: 5px; background-color: rgb(32,32,32);"></span>
> 
> 
> 
> 

Hi

File: blockchains/kavaevm/assets/0x919C1c267BC06a7039e03fcc2eF738525769109c/info.json (hunk 1 above)
Comment (britto10) [reply blockchains/kavaevm/assets/0x919C1c267BC06a7039e03fcc2eF738525769109c/info.json]: Myr

**Pull request merged**